.github/workflows/triage.yml

```diff
@@ -0,0 +1,26 @@
+name: Triage
+
+on:
+  pull_request_target:
+    types: [opened]
+#   issues:
+#     types: [opened]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Request Labeler
+#         if: github.event_name == 'pull_request_target'
+        uses: actions/labeler@v3
+        with:
+          configuration-path: .github/pr-labeler.yml
```

Comment:
Does `or-labeller.yml` still useful ? Because it has been removed in the PR

Reply:
Yes, workflow file`.github/workflows/pr-labeler.yml` is removed, and action moved to `.github/workflows/triage.yml` but `.github/pr-labeler.yml` is a config for that action, containing info about labels. I think we cannot put invalid workflow yml file inside workflows dir, that's why config file lives there.